src/EditorFeatures/Core.Wpf/NavigableSymbols/NavigableSymbolService.NavigableSymbol.cs

```diff
@@ -55,13 +52,16 @@ private async Task NavigateAsync()
             {
                 // we're about to navigate.  so disable cancellation on focus-lost in our indicator so we don't end up
                 // causing ourselves to self-cancel.
-                using var backgroundIndicator = _indicatorFactory.Create(
+                using var backgroundIndicator = _service._backgroundWorkIndicatorService.Create(
                     _textView, SymbolSpan,
                     EditorFeaturesResources.Navigating_to_definition,
-                    cancelOnFocusLost: false);
+                    new()
+                    {
+                        CancelOnFocusLost = false
+                    });
 
                 await _location.TryNavigateToAsync(
-                    _service._threadingContext, new NavigationOptions(PreferProvisionalTab: true, ActivateTab: true), backgroundIndicator.UserCancellationToken).ConfigureAwait(false);
+                    _service._threadingContext, new NavigationOptions(PreferProvisionalTab: true, ActivateTab: true), backgroundIndicator.CancellationToken).ConfigureAwait(false);
```

Comment:
@kayle curious why the change in naming here.  seems inconsistent with the TWD api.